openfga/v1/openfga.proto

```diff
@@ -61,18 +61,7 @@ message Tuple {
 }
 
 message TupleKeys {
-  repeated TupleKey tuple_keys = 1 [
-    (validate.rules).repeated = {
-      min_items: 1,
-      max_items: 25
-    },
-    json_name = "tuple_keys",
-    (google.api.field_behavior) = REQUIRED,
-    (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
-      minimum: 1,
-      maximum: 25
-    }
-  ];
+  repeated TupleKey tuple_keys = 1;
```

Comment:
We should leave the `min_items` validation, yeah? We want to validate that at least one tuple_key was provided, but the max is validated in code based on config settings.

Reply:
I don't know. What if you are writing tuples programmatically you may do something like
```
pb.WriteRequest(Writes: *pb.Tuples{tuples})
```
and tuples may actually be a zero-length array. Do we want to allow this case or should the user check if the array length is > 0?

Reply:
Current behavior is to enforce at least 1 write. The purpose of this PR is mostly to fix the behavioral issue with the configured maximum number of writes. For that reason, let's keep this purposed to that issue. Let's bring up the 0 len write issue with the core team and get an opinion on it 👍

Reply:
Sounds good!